www/webapp/src/views/CrudListDomain.vue

```diff
@@ -27,7 +27,7 @@ export default {
         },
         texts: {
           banner: () => 'To edit your DNS records, click on one of your domains.',
-          create: () => `You have ${self.availableCount} of ${self.limit_domains} domains left.<br /><small>Contact support to apply for a higher limit.</small>`,
+          create: () => self.limit_domains ? `You have ${self.availableCount} of ${self.limit_domains} domains left.<br /><small>Contact support to apply for a higher limit.</small>` : '',
```

Comment:
condition evaluates to false when `limit_domains = 0`; you probably want to check for `null`?